src/integrationTest/java/uk/gov/hmcts/darts/task/runner/impl/CaseExpiryDeletionAutomatedTaskITest.java

```diff
@@ -66,6 +71,36 @@ void positiveRetentionDatePassed() {
         assertCase(caseId1, true);
     }
 
+    @Test
+    void positiveRetentionDatePassedForOneCaseButNotAnotherEventNotAnoymised() {
+        CourtCaseEntity courtCase1 = createCase(-1, CaseRetentionStatus.COMPLETE);
+        CourtCaseEntity courtCase2 = createCase(-1, CaseRetentionStatus.PENDING);
+
+        EventEntity event = dartsDatabase.getEventLinkedCaseRepository().findAllByCourtCase(courtCase1).getFirst().getEvent();
+        eventLinkedCaseStub.createCaseLinkedEvent(event, courtCase2);
+        final int caseId1 = courtCase1.getId();
+
+        caseExpiryDeletionAutomatedTask.preRunTask();
+        caseExpiryDeletionAutomatedTask.runTask();
+
+        assertCase(caseId1, true, event.getId());
+    }
+
+    @Test
+    void positiveRetentionDatePassedForBothCaseLinkedEventsAnoymised() {
```

Comment:
I'm finding some of these integration test names difficult to read. There was some work done by Mario a while back to try and standardise and make clearer test names. See here for examples https://github.com/mario-paniccia/darts-api-fork/pull/1 Any change you could try and incorporate that? e.g. given when then style?

Reply:
My understanding is JUnit5 test names can't have underscores in them as it fails HMCTS style rules.
However I can add a @DislayName to the tests to provide more human readable context

Reply:
I think there's a rule exclusion somewhere to allow underscores in test names. It's how I tend to name my tests nowadays.

Reply:
Is that an exclusion we added as the HMCTS base spring template prevents its? 
(My understanding is we should be following the HMCTs styles)

Reply:
I assume it's this one `<exclude name="MethodNamingConventions"/>` in `config/pmd/test-ruleset.xml`, which was added by us a year ago.

Reply:
If we added this exclusion it will be worth checking with the spring template and HMCTs standards to see why this was added in the first place. As we should be aligned to both the standards and templates

Reply:
That exclusion is only in the test ruleset so in my opinion that is fine if we think it is benficial for our project, and on this occasion I believe that to be the case as it allows us to write much clearer test names than what we have in this PR. 
There are many examples of other HMCTS projects using underscores within test names, we are not the first team to do this. 

For the purpose of this PR, if you don't want to align with what is being suggested, can you at least make the test name clearer? Start by removing this word 'positive' in front of every test and say what the outcome of the test should be, as currently that is not clear.

Reply:
I'm happy to change it but this pattern has been used on almost 100 PRs I have raised (and merged) on darts, why is this only being mentioned now?